tests/samplers_tests/test_grid.py

```diff
@@ -57,9 +51,6 @@ def sorted_values(
     all_suggested_values = [tuple([p for p in sorted_values(t.params)]) for t in study.trials]
     assert set(all_grids) == set(all_suggested_values)
 
-    ids = sorted([t.system_attrs["grid_id"] for t in study.trials])
-    assert ids == list(range(n_grids))
```

Comment:
I think we can remove this because it asserts the implementation details, not an expected behavior of GridSampler.